### PR TITLE
397: add docs

### PR DESCRIPTION
**Pull request by sviatosluv**

### CODE REVIEW

File: lib/chat/sync/cargo_scenario.livemd [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Cargo scenario
+
+## Led indication
+
+### Pins
+
+`GPIO23` -> plug the red led
```
Comment (sergey-lukianov) [inline lib/chat/sync/cargo_scenario.livemd:7]: please add pin numbers

File: lib/chat/sync/cargo_scenario.livemd [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`GPIO25` -> plug the green led
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Flow statuses
```
Comment (sergey-lukianov) [inline lib/chat/sync/cargo_scenario.livemd:13]: describe drivers flow scenario

File: lib/chat/sync/cargo_scenario.livemd [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+1. `Drive Complete`
+
+Indication: no red light, solid green light.
+When the syncronization was succesful and all the data from camera sensors was stored into the system.
+
+1. `Drive Reset`
+
+Indication: no red light, no green light.
+Something wrong with the syncing.
```
Comment (sergey-lukianov) [inline lib/chat/sync/cargo_scenario.livemd:39]: Scenario finished. Ready for the next flash drive

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: lib/chat/sync/cargo_scenario.livemd (hunk 3 above)
Comment (sviatosluv) [reply lib/chat/sync/cargo_scenario.livemd]: done

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**